### PR TITLE
Multi-language docs examples for Kafka Streams

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

File: src/main/docs/guide/kafkaStreams.adoc [hunk 1]
```diff
@@ -109,20 +96,13 @@ The above configuration sets the `num.stream.threads` setting of the Kafka `Stre
 
 You can then inject an `api:configuration.kafka.streams.ConfiguredStreamBuilder[]` specifically for the above configuration using `jakarta.inject.Named`:
 
-[source,java]
-----
-@Named("my-other-stream")
-KStream<String, String> myOtherKStream(@Named("my-other-stream") ConfiguredStreamBuilder builder)  {...}
-----
+snippet::io.micronaut.kafka.docs.streams.WordCountStream[tags="clazz, myOtherStream"]
```
Comment (guillermocalvo) [inline src/main/docs/guide/kafkaStreams.adoc:99]:
```suggestion
snippet::io.micronaut.kafka.docs.streams.WordCountStream[tags="myOtherStream"]
```

Unless you have a specific reason in mind to make this snippet longer than it was, I would leave out `clazz` to keep it similar to the original example:

```java
@Named("my-other-stream")
KStream<String, String> myOtherKStream(@Named("my-other-stream") ConfiguredStreamBuilder builder)  {...}
```

File: src/main/docs/guide/kafkaStreams.adoc [hunk 2]
```diff
@@ -109,20 +96,13 @@ The above configuration sets the `num.stream.threads` setting of the Kafka `Stre
...
 NOTE: If you do not provide a `@Named` on the `ConfiguredStreamBuilder` you have multiple KStreams defined that share the default configurations like client id, application id, etc.It is advisable when using multiple streams in a single app to provide a `@Named` instance of `ConfiguredStreamBuilder` for each stream.
 
 .Kafka Streams Word Count
-[source,java]
-----
-include::{testskafkastreams}/wordcount/WordCountStream.java[tags=namedStream,indent=0]
-}
-----
+
+snippet::io.micronaut.kafka.docs.streams.WordCountStream[tags="clazz, namedStream"]
```
Comment (guillermocalvo) [inline src/main/docs/guide/kafkaStreams.adoc:105]: Same here:

```suggestion
snippet::io.micronaut.kafka.docs.streams.WordCountStream[tags="namedStream"]
```

**Comment by guillermocalvo** [comment]
LGTM 👍

**Comment by sdelamo** [comment]
> LGTM 👍

@guillermocalvo Please, use the GitHub review PR functionality to signal whether you approve, reject or leave comments in a PR.

**Review by sdelamo — APPROVED** [review]

**Comment by guillermocalvo** [comment]
> @guillermocalvo Please, use the GitHub review PR functionality to signal whether you approve, reject or leave comments in a PR.

@sdelamo I left a comment because at the time the PR was not ready for review yet. Do you guys usually approve/reject draft PRs? I can do that next time.

**Comment by sdelamo** [comment]
> > @guillermocalvo Please, use the GitHub review PR functionality to signal whether you approve, reject or leave comments in a PR.
> 
> @sdelamo I left a comment because at the time the PR was not ready for review yet. Do you guys usually approve/reject draft PRs? I can do that next time.

no, we don't review Draft PRs generally.

**Comment by guillermocalvo** [comment]
> no, we don't review Draft PRs generally.

That's what I thought, which is why I left a comment in the draft PR to indicate that it looked good to me so far.

**Review by guillermocalvo — CHANGES_REQUESTED** [review]
`WordCountStreamTest` is satisfied if `client.publishSentence()` doesn't throw any exceptions; I don't believe that's proof enough that the Kafka stream is doing its job, is it?

I'm also missing `WordCountStreamTest` for `test-suite-groovy` and `test-suite-groovy`.

**Review by guillermocalvo — APPROVED** [review]

**Pull request merged**